signals/links/link_all_from_sharepoint.yml

```
@@ -0,0 +1,4 @@
+name: "Link: From notifications.google.com"
```

Title doesn't match the intent. 
```suggestion
name: "Link: From sharepoint.com"
```